--- data/dv/intermediate/ooxml/HFD/4337-12.xml	1980-01-01 00:00:00.000000000 +0100
+++ /tmp/tmp9qjKrQ	2016-10-02 22:33:16.407977000 +0200
@@ -329,7 +329,7 @@
                 <w:sz w:val="16" />
                 <w:szCs w:val="16" />
               </w:rPr>
-              <w:t>4337-12</w:t>
+              <w:t>0000-12</w:t>
             </w:r>
             <w:r w:rsidRPr="00494B59">
               <w:rPr>
