--- data/dv/intermediate/docbook/HDO/B276-03.xml	2010-05-28 06:59:06.248000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpmuvlub	2010-05-28 06:58:43.875000000 +0200
@@ -263,11 +263,11 @@
 i en av B.C. disponerad dator och att bilderna enligt E.G:s uppgifter -
 som tingsrätten sätter tilltro till - tagits av B.C. under den tid
 parterna sammanlevde. B.C. har påstått att bilderna lagts in i datorn
-från en kassett som lämnats till honom av en person vid namn Hans
-Dahlström, som emellertid inte trots efterspaning från polisen kunnat
+från en kassett som lämnats till honom av en person vid namn H.D,
+som emellertid inte trots efterspaning från polisen kunnat
 påträffas och inte heller kunnat återfinnas i befolkningsregistret. Vid
 huvudförhandlingen har B.C. framkastat att mannens namn möjligen är
-Dahlskog. Tingsrätten finner B.C:s uppgifter om den okände mannen icke
+D. Tingsrätten finner B.C:s uppgifter om den okände mannen icke
 förtjäna avseende och anser det ställt utom rimligt tvivel att B.C.
 varit ansvarig för uppsättandet av lapparna. Härigenom har han, såsom
 åklagaren påstått, gjort sig skyldig till sexuellt ofredande mot E.G.
@@ -329,7 +329,7 @@
 dock inte hur man kunde öppna bildfilerna. Vid något tillfälle hade
 någon lagt en diskett i hans brevlåda, och han bad en bekant, ”Ankan”,
 att titta på vad den innehöll. Det var samma bilder som fanns i datorn.
-Han tror att ”Ankan” heter Bo-Anders Dahlskog, men han är inte säker på
+Han tror att ”Ankan” heter B-A. D, men han är inte säker på
 detta. Han hittade vid ett tillfälle i sin trappuppgång och på gården
 ett antal exemplar av svart/vita collagelappar med bilder på E.G. I
 samband med ett polisförhör visade han en sådan lapp för polismannen.
