--- data/dv/intermediate/docbook/HDO/B415-07.xml	2009-09-27 10:02:15.191000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpphn4xa	2009-09-27 10:01:48.019099900 +0200
@@ -585,7 +585,7 @@
 bevisning om saken i de flesta fall. En viss schablonisering i
 bedömningen bör därför tillåtas. Därvid synes det vara rimligt att i
 fall där den yngre än mycket nära 15- årsgränsen godta en större
-åldersskillnad än när den yngre än just över 14 år. Det anförda leder
+åldersskillnad än när den yngre är just över 14 år. Det anförda leder
 till att skillnad i ålder och utveckling i det här aktuella fallet inte
 skall anses så stor att ansvarsfrihetsregeln inte kan tilllämpas.
 
