--- data/dv/intermediate/ooxml/HDO/T3258-09.xml	1980-01-01 00:00:00.000000000 +0100
+++ /tmp/tmpRadTzp	2011-08-14 20:19:14.000000000 +0200
@@ -742,7 +742,7 @@
                 <w:sz w:val="16" />
                 <w:szCs w:val="16" />
               </w:rPr>
-              <w:t>2010-11 30</w:t>
+              <w:t>2010-11-30</w:t>
             </w:r>
             <w:r w:rsidRPr="00494B59">
               <w:rPr>
