--- data/dv/intermediate/docbook/HDO/T3726-93_1.xml	2014-09-01 12:35:13.000000000 +0200
+++ /tmp/tmpTApVkT	2014-09-01 12:35:08.000000000 +0200
@@ -250,7 +250,7 @@
 behållning på 33 kr.
 
 Under den senare delen av äktenskapet levde makarna L på fastigheten
-Trästa 1:10 i Sollefteå. Under 1980 lät de uppföra ett prefabricerat
+X. Under 1980 lät de uppföra ett prefabricerat
 smålins på fastigheten, vilket i huvudsak finansierades genom upptagande
 av statliga lån. M.L. stod som delägare till fastigheten och delade
 betalningsansvaret for lånen med maken, oaktat att hon själv saknade
@@ -263,15 +263,15 @@
 äganderätt. Enligt den efter M.L. upprättade bouppteckningen utgjorde
 hennes andel ca 92 % av dödsboets tillgångar, vilken kvotdel senare
 kommer att ha betydelse jämlikt 3 kap 2 § ÄB vid arvskifte efter L.L..
-Fastigheten Trästa 1:10 har därvid åsatts ett marknadsvärde av 465 000
+Fastigheten X har därvid åsatts ett marknadsvärde av 465 000
 kr. Därutöver erhöll L.L. likvid från tre livförsäkringar, för vilka han
 insatts som förmånstagare med ett sammanlagt belopp om ca 74 000 kr.
 
-Omkring en månad efter M.L:s frånfälle överlät L.L. fastigheten Trästa
-1:10 till döttrarna A-S.L. och A.L.. Överlåtelsen har fått formen av en
+Omkring en månad efter M.L:s frånfälle överlät L.L. fastigheten X
+till döttrarna A-S.L. och A.L.. Överlåtelsen har fått formen av en
 gåva, oaktat att gåvomottagarna övertog inteckningsgäld till ett belopp
 om 341 000 kr och med beaktande av att fastighetens taxeringsvärde då
-uppgick till 201 000 kr. Fastigheten Trästa 1:10 har enligt en i april
+uppgick till 201 000 kr. Fastigheten X har enligt en i april
 1991 företagen värdering ett saluvärde om 465 000 kr +/-20 000 kr.
 Kärandena anser emellertid att värdet är i underkant och menar att
 saluvärdet sannolikt är omkring 750 000 kr. L.L. fortsatte, trots att
@@ -306,7 +306,7 @@
 
 Sammanfattningsvis gör P.A. och A.B. gällande att den
 förmögenhetsöverföring som skett under åren från L.L. till M.L. i
-förening med gåvan av fastigheten Trästa 1:10 lett till ett resultat som
+förening med gåvan av fastigheten X lett till ett resultat som
 är oskäligt mot dem. Grund för jämkning av förmånstagarförordnande
 föreligger därför.
 
@@ -393,7 +393,7 @@
 visar att de inte är beroende av försäkringsbeloppet för sin
 försörjning. Parternas ekonomi utgör alltså i och för sig ej skäl för
 jämkning. Emellertid visar utredningen att L.L. genom äktenskapsförord,
-inbördes testamente och överlåtelse av fastigheten Trästa 1:10 vidtagit
+inbördes testamente och överlåtelse av fastigheten X vidtagit
 förmögenhetsöverföringar, vilka ensidigt gynnat A-S.L. och A.L. i
 förhållande till deras halvsystrar. Enligt det preliminära arvskiftet
 erhåller således P.A. och A.B. knappt 3 000 kr av en total bobehållning
@@ -560,7 +560,7 @@
 boet efter M.L. har testamentsvillkoret om rätt till laglott i stället
 för arvslott haft endast mindre betydelse för dem. Av samma skäl har
 deras rätt till arv efter L.L. inte heller i nämnvärd utsträckning
-påverkats av dennes gåva av fastigheten Trästa 1:10 till A-S.L. och
+påverkats av dennes gåva av fastigheten X till A-S.L. och
 A.L., för det fall att denna gåva inte har lagts samman med
 kvarlåtenskapen i övrigt med stöd av 7 kap 4 § ÄB.
 
@@ -686,4 +686,4 @@
 
  </chapter>
 
-</book>
\ Ingen nyrad vid filslut
+</book>
