--- data/dv/intermediate/ooxml/MÖD/M2557-10.xml	1980-01-01 00:00:00.000000000 +0100
+++ /tmp/tmpis2E97	2011-08-14 20:19:19.000000000 +0200
@@ -500,7 +500,7 @@
                 <w:sz w:val="16" />
                 <w:szCs w:val="16" />
               </w:rPr>
-              <w:t>2011 03-23</w:t>
+              <w:t>2011-03-23</w:t>
             </w:r>
             <w:r w:rsidRPr="00450E68">
               <w:rPr>
