--- data/dv/intermediate/ooxml/HON/B379-14.xml	1980-01-01 00:00:00.000000000 +0100
+++ /tmp/tmpMUv_My	2017-01-18 15:37:29.630386000 +0100
@@ -1434,8 +1434,8 @@
               personer, om det varit fråga om gemensamt våld mot en ensam person eller om
               våldsutövningen fortsatt efter det att den utsatte blivit försvarslös (se ovan
               nämnda rättsfall). Det faktum att K hållit ett spädbarn i famnen under attacken
-              är självfallet en mycket speciell omständighet. Främst därför bör Simon
-              Engelmark anses ha visat särskild hänsynslöshet, vilket också är ett kriterium
+              är självfallet en mycket speciell omständighet. Främst därför bör S
+	      anses ha visat särskild hänsynslöshet, vilket också är ett kriterium
               för att bedöma misshandeln som grovt brott.</w:t>
             </w:r>
           </w:p>
@@ -1581,7 +1581,7 @@
               </w:rPr>
               <w:lastRenderedPageBreak />
               <w:t>bedöma misshandeln som synnerligen grov (jfr rättsfallet NJA 2011 s. 89).
-              Simon Engelmark ska därför dömas för grov misshandel.</w:t>
+              S ska därför dömas för grov misshandel.</w:t>
             </w:r>
             <w:r w:rsidR="00C82D87" w:rsidRPr="00494B59">
               <w:rPr>
